package.json

```diff
@@ -15,11 +15,13 @@
     "@material-ui/core": "^4.8.2",
     "@material-ui/icons": "^4.5.1",
     "express": "^4.17.1",
+    "file-loader": "^5.0.2",
```

Comment:
why was file-loader added?

Reply:
It's for importing the map marker png image.

Reply:
ok thanks!